packages/office-ui-fabric-react/src/components/Facepile/__snapshots__/Facepile.test.tsx.snap

```diff
@@ -194,18 +277,22 @@ exports[`Facepile renders Facepile correctly 1`] = `
               font-size: 14px;
               font-weight: 400;
               outline: transparent;
-              padding-bottom: 0;
-              padding-left: 16px;
-              padding-right: 16px;
-              padding-top: 0;
+              padding-bottom: 0px;
+              padding-left: 0px;
+              padding-right: 0px;
```

Comment:
Are you sure we want to accept the styling change of padding horizontal from 16px to 0px?

Reply:
I'm not sure where it got the 16px value from, but the value was 0px in the scss file so this output is what I would expect. So I'm not sure why this test is reflecting a change, it seems to be the same as before. But this 0px value should be correct as far as I can tell.